.github/workflows/integration_tests.yaml

```diff
@@ -0,0 +1,26 @@
+name: Integration Tests
+
+on: pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  integration_tests:
+    name: Integration tests
+    uses: uc-cdis/gen3-code-vigil/.github/workflows/integration_tests.yaml@master
```

Comment:
why are we using this workflow? https://github.com/uc-cdis/gen3-code-vigil/blob/master/.github/workflows/integration_tests.yaml

shouldn't it be either https://github.com/uc-cdis/.github/blob/master/.github/workflows/integration_tests.yaml
or https://github.com/uc-cdis/gen3-code-vigil/blob/master/.github/workflows/shared_integration_tests.yaml?
(btw, we should delete one of the 2 to avoid confusion)

Reply:
The script in gen3-code-vigil is used within that repo, so changes to the workflow get tested right there. Once it works fine, we copy it over to .github repo and that is the one used across other repos.

Reply:
i see, i still think we should remove one, you can still test changes 🤔